kingpin/actors/rightscale/server_array.py

```diff
@@ -705,6 +706,12 @@ def _execute(self):
         # Finally, monitor all of the tasks for completion.
         self.log.info('Queueing %s tasks' % count)
         task_waiting = []
+
+        import os
+        os.environ['TZ'] = 'UTC'
+        time.tzset()
+        tasks_start = time.strftime('%Y/%m/%d %H:%M:%S +0000')
```

Comment:
Comments here about why you're doing this. I know why today, but i won't remember why in 2 weeks.